src/include/miopen/solver_id.hpp

```diff
@@ -61,7 +61,8 @@ enum class Primitive
     Adam,
     Item,
     RoPE,
-    ReLU
+    ReLU,
+    Pdist
```

Comment:
update code with branch develop-moreh

Reply:
solved

This happened because I initially checked out the wrong branch (`develop`) when starting to write the new operation—it should have been `develop-moreh`. I resolved the issue by merging the changes with `develop-moreh` and resolving all conflicts.